spec/models/post_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```

Comment:
- [ ] I suggest that any custom methods in the models should have a corresponding unit test. So make sure that you add a test for the methods `update_posts_counter` and `recent_comments` in your Post model. :blush: